cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -437,9 +438,20 @@ private WorkflowModellingResult compileWorkFlow(List<Map<String, Map<String, Obj
         return new WorkflowModellingResult(new Workflow(steps), errors);
     }
 
+    private String getStepName(Map<String, Map<String, Object>> stepRawData, List<RuntimeException> errors) {
+        String stepName = null;
+        try {
+            stepName = stepRawData.keySet().iterator().next();
+            executableValidator.validateStepName(stepName);
```

Comment:
maybe we should have a convention for validators that all of them throw exceptions or all of them have an errors list as a parameter which is returned by the validators, WDYT?